.circleci/config.yml

```diff
@@ -382,7 +382,7 @@ jobs:
           name: Fetch CocoaPods Specs
           command: |
             Y | sudo gem uninstall cocoapods
-            sudo gem install cocoapods -v 1.5.3
+            sudo gem install cocoapods -v 1.9.1
```

Comment:
has this line been tested in our CircleCI pipeline?

Reply:
@antonioned will try to hit it once merged to this target branch of pure RN, but exactly same pod and XCode versions were used locally as defined in CircleCI config